### DISCUSSION TITLE
Bot Usability

### DISCUSSION THREAD

**Discussion by devparanjay** [nerap/McTrade#43] [General]
Hello, looks like a really cool bot!
I wanted to know whether I can start using the bot or if it is still in the initial development phase?

**Comment by nerap** [comment]
Hi, so actually you can use it, but I don't recommend too.
It will trade but it's not actually the way that I want the bot to trade, I need to make some adjustement, but don't worry it's a matter of days 👍

**Reply by devparanjay** [reply]
Oh alright, cool!
I kind of already installed it but now I'm waiting up for the update!
I was looking for a bot that can just make small profits every day with low-risk with as many or as few trades as necessary - a bot I can just leave running all day. I then found out McTrade and was just about to test it with some crypto but then thought to first confirm whether the bot is usable or not.
Looks like a really cool project, will you please update me here when the update is out? Also, it'll be great if you could explain the bot's trading strategy/process in the ReadMe. 😁

**Reply by nerap** [reply]
You can use it now 😄

**Reply by devparanjay** [reply]
Yay! I'm really excited, thanks! :)
